tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -95,6 +136,77 @@ const QuirkList = (props: {
+                        popperContent={
+                          <Box>
+                            {!!quirk.customization_options &&
+                              hasExpandableCustomization && (
+                                <Box
+                                  mt="1px"
+                                  style={{
+                                    'box-shadow':
+                                      '0px 4px 8px 3px rgba(0, 0, 0, 0.7)',
+                                  }}>
+                                  <Stack
+                                    onClick={(e) => {
+                                      e.stopPropagation();
+                                    }}
+                                    maxWidth="300px"
+                                    backgroundColor="black"
+                                    px="5px"
+                                    py="3px">
+                                    <Stack.Item>
+                                      <PreferenceList
+                                        act={act}
+                                        preferences={getCorrespondingPreferences(
+                                          quirk.customization_options,
+                                          data.character_preferences
+                                            .manually_rendered_features
+                                        )}
+                                        randomizations={getRandomization(
+                                          getCorrespondingPreferences(
+                                            quirk.customization_options,
+                                            data.character_preferences
+                                              .manually_rendered_features
+                                          ),
+                                          props.serverData,
+                                          props.randomBodyEnabled,
+                                          props.context
+                                        )}
+                                        maxHeight="100px"
+                                      />
+                                    </Stack.Item>
+                                  </Stack>
+                                </Box>
+                              )}
+                          </Box>
+                        }>
+                        <Button
+                          disabled={!props.selected}
+                          selected={customizationExpanded}
+                          icon="cog"
+                          tooltip={
+                            props.selected
+                              ? customizationExpanded
+                                ? 'Click this button to close the customization menu!'
+                                : 'This quirk is customizable! Click this button to open a customization menu!'
+                              : 'You must take this quirk before you can customize it!'
```

Comment:
UX tips:

1. Some of these are too verbose
2. You don't need a tooltip to say close

I would make the tooltip say "Customize", nothing more. Then, I wouldn't even show the button if the quirk isn't selected.